docs/PROJECT.asciidoc

```
@@ -0,0 +1,73 @@
+The goal of this document is to provide a list of items required when
+working on scoping a project. It uses the same steps laid out in the
+experiment doc but the wording is aimed at improving project scoping
+and success criteria.
+
+== Title:
+
+=== Problem Statement
+
+* What question(s) are you trying to answer? Limit the scope where possible.
+* Include background information, observations, and evidence surrounding the
+  problem space.
+* What business impact does this have?
+
+=== Suggested Improvement (Project Plan)
+
+* State a possible solution for the problem at hand.
+* Identify success criteria
+* Refer to documentation that supports this method
```

Also what are possible recalibration actions at each major stage/phase of the project. When can be "fail fast" (I know, I know, I couldn't resist saying it).